playbooks/ai-on-openlab/pre.yaml

```diff
@@ -0,0 +1,93 @@
+- name: AI On Openlab Pre Job
+  hosts: all
+  become: yes
+  tasks:
+    - name: Set Default Hardware type Parameters
+      set_fact:
+        user_file: "user_data"
+      when: not is_gpu_enabled
+
+    - name: Split Hardware type Parameters for GPU
+      set_fact:
```

Comment:
please to use variable to make things simpler: 

vars:
user_file: '{{ is_gpu_enabled | bool | ternary("user_data_gpu", "user_data") }}'